internal/cmd/start/start.go

```diff
@@ -17,15 +17,15 @@ var flags = []cli.Flag{
 		Aliases: []string{"a"},
 		Usage:   "host listen address",
 		Value: cli.NewStringSlice(
-			"/ip4/0.0.0.0/udp/2020/quic",
-			"/ip6/::0/udp/2020/quic"),
+			"/ip4/0.0.0.0/udp/0/quic",
```

Comment:
Previously multiple `ww start` commands raised an error, because they all used the same listening port.